Automattic/wp-calypso
# A4A: Add scrolling to the DataViews view table/list and into the PreviewPane tabs

Resolve: https://github.com/Automattic/automattic-for-agencies-dev/issues/117

## Proposed Changes

The scroll in the Activity tab:

![image](https://github.com/Automattic/wp-calypso/assets/9832440/5f73b6e0-ec49-4638-a42a-9d434b90e720)


## Testing Instructions

- Check the code
- If you have more than 3 sites, reduce the browser screen height, now you can scroll the Sites table view. (Note: it will scroll the table header, this is a temporary fix until the DataViews component (Gutenberg package fixes it)
- Select a site (with Jetpack features activated), it will open the Preview Pane and check the scroll in the Activity tab
- Reduce the browser screen height, now you can scroll the Sites list view (the side-by-side left bar)

## Pre-merge Checklist
